packages/firebase_core/firebase_core/ios/firebase_core.podspec

```diff
@@ -27,7 +27,7 @@ Pod::Spec.new do |s|
   s.source_files     = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
 
-  s.ios.deployment_target = '9.0'
+  s.ios.deployment_target = '11.0'
```

Comment:
I see on https://github.com/firebase/firebase-ios-sdk/blob/174be0d92bd6706b30a7a9d18f6bbe2319809e03/Firebase.podspec that the ios.deployment_target is still 10.0.

Do we need to manually add the deployment target? Or should CocoaPod just pick it from the dependency? It would me one less thing to maintain from our side

Reply:
I see what you mean, but see what @Salakar thinks as I'm not sure.

Reply:
I believe we can remove it or at a minimum set it back to 10.0

Reply:
Removing deployment targets caused [build failures](https://github.com/firebase/flutterfire/actions/runs/3264629221) so I guess we do need them. Not sure we should go back to version 10, @Salakar.  [See documentation](https://firebase.google.com/support/release-notes/ios#version_1000_-_october_10_2022). Also - look at the CI logs. Sample:

```
of supported deployment target versions is 11.0 to 16.0.99. (in target 'FirebaseAppCheckInterop' from project 'Pods')
    /Users/runner/work/flutterfire/flutterfire/tests/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 10.0, but the range of supported deployment target versions is 11.0 to 16.0.99. (in target 'GoogleAppMeasurement' from project 'Pods')
```

Reply:
The pod relies on 10.0 and above still, but if the docs say they want to support 11+ then sticking to 11 here is fine by me!

Reply:
I guess this makes it a breaking change now across all plugins. I'm also a little concerned about the behaviour of Storage (it's completely changes to a Swift implementation) as I wonder what other edge cases have changed (e.g. exceptions/errors) that we aren't capturing in our CI.